cadasta/test/conftest.py

```diff
@@ -85,6 +87,48 @@ def any_org_member(request, all_fixtures):
                 if request.param in user['username'])
 
 
+@pytest.fixture(scope='session',
+                params=['functest_generic_',
+                        'functest_genericphone_',
+                        'functest_org_admin_',
+                        'functest_org_member_',
+                        'functest_prj_manager_',
+                        'functest_data_collector_',
+                        'functest_prj_user_'])
+def any_user(request, all_fixtures):
```

Comment:
Could you add a docstring here and explain what happens in this function?

Reply:
Done.